js/defs.bzl

```diff
@@ -41,7 +39,12 @@ def js_test(**kwargs):
         **kwargs
     )
 
-js_package = _js_package
+def js_package(**kwargs):
+    # Default the version to "0.0.0" for the public API
+    version = kwargs.pop("version", "0.0.0")
```

Comment:
maybe we should require it explicit for now?

Reply:
the implicit is nice since it really isn't load bearing on the happy path for first party deps; but I don't like it in this macro as it doesn't make it into docs so I'll move it to the rule